docker-compose.yml

```diff
@@ -1,16 +1,15 @@
 version: "3.4"
 
 services:
-  localstack:
-    image: localstack/localstack:latest
+  s3:
+    image: minio/minio:latest
+    command: server /data
     environment:
-      - SERVICES=s3
-      - PORT_WEB_UI=8081
-      - DEFAULT_REGION=eu-west-2
-      - AWS_ACCESS_KEY_ID=foo
-      - AWS_SECRET_ACCESS_KEY=bar
+      MINIO_ACCESS_KEY: longkey
+      MINIO_SECRET_KEY: verysecretkey
+      MINIO_REGION: eu-west-2
     ports:
-      - "8081:8081"
+      - 9000:9000
```

Comment:
this is a YAML string, so whenever there is any non-alphanumeric character keep the `"` quotes around it for safety:
https://twitter.com/sistasonDE/status/1045353434394841088